README.md

```diff
@@ -11,7 +11,7 @@ https://rogerdudler.github.io/git-guide/
 
 ## Agenda
 Prerequisites: Please come with Git already installed.
-TBD
+Timings: 9am to 5pm
```

Comment:
this is what I showed as a demonstration in the video. Please read the task carefully. Look at the other merged PRs